Arrays/DynamicArray.cpp

```diff
@@ -257,40 +254,196 @@ void DynamicArray::Reverse()
     
 }
 
-void DynamicArray::Swap(int *x, int *y)
+void DynamicArray::Swap(int x, int y)
 {
     int temp;
-    temp = *x;
-    *x = *y;
-    *y = temp;
+    temp = A[x];
+    A[x] = A[y];
+    A[y] = temp;
 }
```

Comment:
For this function to become user-friendly in the switch case I had to change it. Refer to line 296 in PR 
`cout<<"18 Swap elements in the array at given indices"<<endl;`